### ISSUE TITLE
collectionViewCell flickers/blinks

### ISSUE THREAD

**Issue by JARMourato** [terminatorover/RGCardViewLayout#10]
Hi, i've verified that when you slide (slowly) the appearing view  (on the right for example) flickers/blinks a bit before it animates smoothly. Can you fix this ? 

PS:  I don't know if I made myself clear, to test, slide very slowly and observe the sections views appearing ; UPDATE: actually, in your demo, I found that the issue is only observable in landscape mode.

**Comment by SohailKh** [comment]
it seems like its an issue with the width of the card. if i size my card down, i don't get that issue.

**Comment by JARMourato** [comment]
How much do you have to size it down so the issue disappears ?

**Comment by SohailKh** [comment]
Not sure the exact amount. I sized mine down about 30%. In any case, it seems to randomly despite the size. Sometimes when I drag slowly I see it, sometimes when I swipe too quickly.

I think it might be an issue with uicollectionview recognizing when the sections are changing? Thoughts?

**Comment by JARMourato** [comment]
I think it might be related to moment when the 3Dtransformation starts to be applied or how it is applied

**Comment by SohailKh** [comment]
from what i can tell, the collectionview drawing the next cell as the previous one is leaving the screen. however, due to the shape and position of the next cell, we're able to see it flickering since the next cell technically should enter before the previous one starts leaving.

if we could potentially load the next cell on drag start or fool the collectionview into loading the next cell earlier, i think we'd be able to solve this problem.

**Comment by KittenYang** [comment]
@SohailKh @JARMourato  Hey,do u guys fix it?

**Comment by SohailKh** [comment]
nope :( drove me crazy.
ᐧ

**Comment by amayne** [comment]
I was able to get rid of this flicker using @KittenYang's comment in another issue:

> Just set the t.m34 to = 1.0/-800

Setting the denominator to a large value for the 3d transform seems to get rid of this for me

**Comment by KittenYang** [comment]
@amayne yeah,it's me. This workaround can fix the issue of visibility of next or prior cell,but it can not fix the flickers/blinks . I've tried this.

**Comment by markhao-cb** [comment]
Any solution for this guys?
